R/export_pacta_files.R

```diff
@@ -162,6 +162,20 @@ export_pacta_files <- function(
   )
   saveRDS(object = iss_emissions, file = iss_emissions_path)
 
+  # Note that this writes to CSV, not RDS.
+  entity_ids_path <- file.path(
+    export_dir,
+    "factset_entity_ids.csv"
+  )
+  logger::log_info("Fetching Factset entity IDs.")
+  entity_ids <- get_entity_ids(
+    conn = conn
+  )
+  logger::log_info(
+    "Exporting FactSet Entity IDs to ", entity_ids_path
+  )
+  write.csv(x = entity_ids, file = entity_ids_path)
```

Comment:
Super nit: you may want to use `row.names = FALSE` and/or `na = ""`, but probably